models/model.js

```diff
@@ -38,4 +38,20 @@ exports.fetchArticlebyIdFromDB = (id) => {
         }
         else return results.rows[0];
     })
+}
+
+exports.fetchCommentsByIdFromDB = (id) => {
+    const queryString = `
```

Comment:
Love that you've seperated out the query string, this makes things super easy to read!